src/components/paragraphs/sum-top-banner/sum-top-banner-paragraph.tsx

```diff
@@ -51,7 +52,7 @@ const SumTopBannerParagraph = ({paragraph, pageTitle, ...props}: Props) => {
           {paragraph.sumTopBannerLink?.url && (
             <div className="rs-pt-2 order-4">
               <ActionLink
-                className="btn block w-fit rounded-full bg-digital-red px-12 py-6 text-5xl font-normal text-white no-underline transition hocus:text-white hocus:underline"
+                className="btn rs-mt-5 flex w-fit items-center rounded-full border-2 border-transparent bg-digital-red px-8 py-4 font-normal text-white transition hocus:border-white hocus:text-white"
```

Comment:
The action link already has some these classes. this is redundant.
https://github.com/SU-SWS/summer-nextjs/blob/1.x/src/components/elements/action-
link.tsx#L18

Reply:
Oh goodness, I do recall us removing that a few PRs ago.

![image](https://media.tenor.com/muj4RR_sPl0AAAAM/homer-simpson-les-simpson.gif)